docs/install_guide_ova.md

```diff
@@ -0,0 +1,114 @@
+3. Follow the installer prompts to perform basic configuration of the appliance and to select the vSphere resources for it to use. 
+    
+    - Accept or modify the appliance name
+    - Select the destination datacenter or folder
+    ![Screenshot of appliance name](img/ovainstall/namelocation.png)
+     - Select the destination host, cluster, or resource pool
+    ![Screenshot of resoure pool](img/ovainstall/resource.png)
+    - Select the disk format and destination datastore
+    ![Screenshot of datastore](img/ovainstall/datastore.png)
+    - Select the network that the appliance connects to
+    ![Screenshot of network](img/ovainstall/network.png)
+
+4. On the **Customize template** page, under Harbor Configure, select the authentication mode and Harbor admin password, if authentication mode is set to ldap_auth, the Harbor LDAP configure is required. the Harbor Admin Password is required, it is used in step 9 to login.
+    ![Screenshot of customize harbor](img/ovainstall/customizeharbor.png)
+    If the Authentication Mode is set to ldap_auth, need to config Harbor LDAP settings:
+    - LDAP Base DN: The base DN to lookup user
+    - LDAP UID: The attribute field to match user, such as uid, cn, email or other attributes.
+    - LDAP URL: The url for LDAP endpoint
+    - Search DN: The user's DN who has the permission to search the LDAP Server
+    - Search DN Password: The password of search DN.
+    - Search Filter: The filter to search users, can be empty.
+    - Search Scope: The scope to search users.
+    ![Screenshot of customize ldap](img/ovainstall/customizeldap.png)
+
+5. On the **Customize template** page, under **System**, set the root password for the appliance VM and the option for **Permit Root Login**. 
+
+    Setting the root password for the appliance is mandatory. 
+
+    **IMPORTANT**: You require SSH access to the vSphere Integrated Containers appliance to perform upgrades. You can also use SSH access in exceptional cases that you cannot handle through standard remote management or CLI tools. Only use SSH to access the appliance when instructed to do so in the documentation.
+    ![Screenshot of customize template system](img/ovainstall/system.png)
+
+6. Expand **Networking Properties** and optionally configure a static IP address for the appliance VM. 
+
+    To use DHCP, leave the **Networking Properties** blank.
+
+    **IMPORTANT**: If you set a static IP address for the appliance, use spaces to separate DNS servers. Do not use comma separation for DNS servers. 
+
+    - Leave the text boxes blank to use auto-generated certificates.
+   
+7. When the deployment completes, refresh the current page and power on the appliance VM.
+   ![Screenshot of power on](img/ovainstall/poweron.png)
+   Press Enter to login
+   ![Screenshot of login root](img/ovainstall/poweron2.png)
+
+   Login to the console with root username and password, and get the harbor appliance address by ifconfig.
```

Comment:
just get the FQDN from the summary page, no need to login to the OVA.

Reply:
Remove login 
Add:
Go to the **Summary** tab for the appliance VM and note the DNS Name.